pkgs/development/libraries/tiledb/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, zlib
+, lz4
+, bzip2
+, zstd
+, spdlog_0
+, tbb
+, openssl
+, boost
+, libpqxx
+, clang-tools
+, catch2
+, python
+, gtest
+}:
+
+stdenv.mkDerivation {
+  name = "tiledb";
+  version = "1.6.2";
```

they've released a new version
```suggestion
  version = "1.6.3";
```

---

Done.